### PR TITLE
ADD: UI for import wallet with passphrase

### PR DESCRIPTION
**Pull request by limpbrains**
It uses alert with input to ask for a passphrase
Currently it works with standard HD wallets + aezeed + slip39

https://user-images.githubusercontent.com/155891/126078571-5bd40b78-367f-45ff-9825-8b3637447b28.mp4

### CODE REVIEW

**Comment by ncoelho** [comment]
I would do the alert before closing the view.

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/73w3fwwypybvqt2xxz1hr3f1jg?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/7340)

**Comment by marcosrdz** [comment]
Same comment as @ncoelho

**Comment by marcosrdz** [comment]
Unless its modified so cancelling passphrase entry sends you back the import screen with the seed filled out

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/mp17r9aje6v642ec45trxd108w?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/7343)

**Comment by limpbrains** [comment]
I've spliced `processImportText` in 2 functions. Now the first one is asking and validating password, if needed, and the second one does the import

https://user-images.githubusercontent.com/155891/126302334-3d5a0213-dadd-4fad-81d3-ded75d68daf5.mp4

Now import can be cancelled and view closes after the alert

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/a6yj6p1tc0dj1gy1qgq2t509nr?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/7351)

**Review by Overtorment — APPROVED** [review]
very nice and clean diff.
good to merge if @ncoelho is onboard product-wise

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/qp0y5a2pegu5qe4z20pzytqpy0?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/7354)

**Review by ncoelho — APPROVED** [review]

File: tests/e2e/bluewallet.spec.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ jasmine.getEnv().addReporter({
 });
 
 describe('BlueWallet UI Tests', () => {
-  it('selftest passes', async () => {
+  it.skip('selftest passes', async () => {
```
Comment (Overtorment) [inline tests/e2e/bluewallet.spec.js:11]: whats with the skips..?

**Review by Overtorment — CHANGES_REQUESTED** [review]

File: tests/e2e/bluewallet.spec.js (hunk 1 above)
Comment (limpbrains) [reply tests/e2e/bluewallet.spec.js]: To speed up the e2e tests by 10000%
fixed

Comment (Overtorment) [reply tests/e2e/bluewallet.spec.js]: the build itself is building ~30min  :-(

**Review by Overtorment — APPROVED** [review]
@marcosrdz

**Review by marcosrdz — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/uq9gxe9xhhzmym0mr4y9huehxc?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/7356)

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**